arrow-mtl-data/src/test/kotlin/arrow/mtl/EitherTTest.kt

```diff
@@ -35,15 +35,15 @@ import arrow.mtl.extensions.eithert.functor.functor
 import arrow.mtl.extensions.eithert.monad.monad
 import arrow.mtl.extensions.eithert.monadError.monadError
 import arrow.mtl.extensions.eithert.traverse.traverse
-import arrow.test.UnitSpec
-import arrow.test.generators.genK
-import arrow.test.generators.throwable
-import arrow.test.laws.AlternativeLaws
-import arrow.test.laws.ConcurrentLaws
-import arrow.test.laws.DivisibleLaws
-import arrow.test.laws.MonadErrorLaws
-import arrow.test.laws.TraverseLaws
-import arrow.test.laws.throwableEq
+import arrow.core.test.UnitSpec
+import arrow.core.test.generators.genK
+import arrow.core.test.generators.throwable
+import arrow.core.test.laws.AlternativeLaws
+import arrow.core.test.laws.ConcurrentLaws
```

Comment:
```suggestion
import arrow.fx.test.laws.ConcurrentLaws
```

and it's necessary to add `arrow-fx-test` dependency.